### ISSUE TITLE
Search is duplicating content

### ISSUE THREAD

**Issue by lbulgarelli** [MIT-LCP/physionet-build#663]
The search is currently duplicating content if a keyword is contained more than once on one of the fields we use for filtering.

A simple `distinct()` would solve the problem, but it is returning an error for some reason.

I don't know when this started happening but it does not seem to be affecting production somehow.

**Comment by bemoody** [comment]
"challenge" returns demobsn once

"eicu" returns demoeicu three times

"database" returns demoeicu three times and demobsn once

So is there something strange about the demoeicu project?

**Comment by elfeto** [comment]
This seems fixed. Maybe #779 ?

**Comment by bemoody** [comment]
No, it's not fixed.  For me, http://localhost:8000/content/?topic=eicu
returns the same result three times.

**Comment by elfeto** [comment]
Maybe is database related, for example https://physionet.org/content/?topic=eicu returns only one.
